### ISSUE TITLE
Include a detailed Readme with install guide + dependencies

### ISSUE THREAD

**Issue by deltaformation** [adamhammes/kijiji-housing#13]
Hello,

It would be helpful to have a quick install guide with required dependencies.

Very interested in trying this project.

Thanks,

**Comment by adamhammes** [comment]
Good idea! Are you more interested in the frontend code or the scraper setup?
